__tests__/server/utils/pollModuleMap.spec.js

```diff
@@ -350,22 +263,34 @@ describe('pollModuleMap', () => {
     loadModulesPromise = Promise.reject(error);
     await pollModuleMap();
     expect(loadModules).toHaveBeenCalledTimes(1);
+    expect(console.error).toHaveBeenCalledTimes(1);
+    expect(console.error).toHaveBeenCalledWith('pollModuleMap: error polling', error);
+
+    expect(setTimeout).toHaveBeenCalledTimes(1);
+    expect(setTimeout).toHaveBeenCalledWith(pollModuleMap, 5e3);
 
-    return loadModulesPromise
-      // catch and then so as to run after pollModuleMap's error handling
-      .catch((err) => err)
-      .then(() => {
-        expect(console.error).toHaveBeenCalledTimes(1);
-        expect(console.error).toHaveBeenCalledWith('pollModuleMap: error polling', error);
-
-        expect(setTimeout).toHaveBeenCalledTimes(1);
-        expect(setTimeout).toHaveBeenCalledWith(pollModuleMap, 5e3);
-
-        expect(setGauge).toHaveBeenCalledTimes(1);
-        expect(setGauge).toHaveBeenCalledWith(holocronMetrics.moduleMapPollWait, 5);
-        expect(incrementGauge).toHaveBeenCalledTimes(1);
-        expect(incrementGauge).toHaveBeenCalledWith(holocronMetrics.moduleMapPollConsecutiveErrors);
-      });
+    expect(setGauge).toHaveBeenCalledTimes(1);
+    expect(setGauge).toHaveBeenCalledWith(holocronMetrics.moduleMapPollWait, 5);
+    expect(incrementGauge).toHaveBeenCalledTimes(1);
+    expect(incrementGauge).toHaveBeenCalledWith(holocronMetrics.moduleMapPollConsecutiveErrors);
```

Comment:
check that `resetPollTime` was called as well

Reply:
this is what the setTimeout check is doing - https://github.com/americanexpress/one-app/pull/962/files#diff-82cec72196769195fc18f858b882683ecae08a9f5fc3b7d097fae4f60d33b404R270